### ISSUE TITLE
how to implement a daily wallpaper changer in react native ( run task even when the app is closed ). 

### ISSUE THREAD

**Issue by zakirtest123** [meharbhutta/react-native-manage-wallpaper#18]
please need your complete guidance/support on how to implement a daily wallpaper changer in react native ( run task even when the app is closed ).

**Comment by Alan-Graton** [comment]
> please need your complete guidance/support on how to implement a daily wallpaper changer in react native ( run task even when the app is closed ).

If you're using Expo you can use the `TaskManager` lib. It's purposes it's to run taks while the app is in background. I'm building a similar app and planning using this lib but Ive not reached this development step yet. Hope this helps ^-^

**Comment by one-dhiraj** [comment]
> > please need your complete guidance/support on how to implement a daily wallpaper changer in react native ( run task even when the app is closed ).
> 
> If you're using Expo you can use the `TaskManager` lib. It's purposes it's to run taks while the app is in background. I'm building a similar app and planning using this lib but Ive not reached this development step yet. Hope this helps ^-^

Yeah i tried that but the Expo `background-fetch` doesn't execute the tasks when the app is killed. Also it is pretty irregular on preiodically executing the task even when the app is actively running. I have since moved my project to react-native-cli. Will update here if i'm able to achieve it using other 3rd party libraries.
